src/controller/ShowsController.ts

```diff
@@ -0,0 +1,54 @@
+import { getShowsByDateDTO } from './../types/getShowsByDateDTO';
+import { ShowsBusiness } from '../bussiness/ShowsBusiness';
+import { showsInputDTO } from './../types/showsInputDTO';
+import { Request, Response } from 'express';
+
+
+
+export class ShowsController {
+    constructor(private showsBusiness: ShowsBusiness) { }
+
+    public async createShow(req: Request, res: Response): Promise<void> {
+        try {
+            const { band_id, week_day, start_time, end_time } = req.body;
+
+            const authenticator = req.headers.authorization as string;
+
+            const show: showsInputDTO = {
+                band_id,
+                week_day,
+                start_time,
+                end_time,
+                authenticator
+            };
+
+            await this.showsBusiness.show(show);
+
+            res.status(200).send({ message: "Show successfully scheduled!" });
```

Comment:
Nesse caso a função precisa ser uma arrow function quando utilizamos o this, para referenciar algo na classe.
Por algum motivo o thi snão funciona em funções normais dentro da classe, tem alguns materiais sobre isso: https://blog.da2k.com.br/2019/01/07/javascript-tudo-sobre-arrow-functions/